dwa_local_planner/src/dwa_planner_ros.cpp

```diff
@@ -518,15 +520,39 @@ namespace dwa_local_planner {
   void DWAPlannerROS::actuator_position_callback(const lexxauto_msgs::ActuatorStatus::ConstPtr& msg)
   {
     this->is_actuator_connect_ = msg->connect;
+
+    // The decision should be delegated to the node responsible for Cargo Status at some point. => AMRCS-174
+    this->is_cargo_fixed_ = false;
+    if (this->cargo_mode_ == "wani")
+    {
+      if (actuator_status::ACT_MID2 <= msg->position[0])
```

Comment:
It is largely a matter of preference, but basically I feel that it is easier to read a variable on the left and a threshold on the right.
好みが大きいですが、基本的には左が変数で右に閾値の方が読みやすく感じます。
```suggestion
      if (msg->position[0] >= actuator_status::ACT_MID2)
```

Reply:
個人的には、不等号は絶対に左辺が小さい値を書く派で、 > と >= はシンタックスエラーにして欲しいくらい読みづらく、
許されるならば全ての不等号の向きを変える作業をしたいくらいなので、このままで行かせて下さい。
https://qiita.com/kitoo/items/bec038105471eec82689